Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py

```diff
@@ -83,9 +89,14 @@ def _mask(ws, masked_ws):
 
     def PyInit(self):
 
-        self.declareProperty(MultipleFileProperty('Run', extensions=['nxs']),
+        self.declareProperty(MultipleFileProperty('Run', action=FileAction.OptionalLoad,
+                                                  extensions=['nxs'], allow_empty=True),
                              doc='File path of run(s).')
 
+        self.declareProperty(MatrixWorkspaceProperty('InputWorkspace', '', direction=Direction.Input,
```

Comment:
Always add the new properties at the very end; there might be user scripts that invoke the algorithm with positional arguments.